lnutil/keyfile.go

```diff
@@ -60,9 +60,13 @@ func LoadKeyFromFileArg(filename string, pass []byte) (*[32]byte, error) {
 	}
 
 	if len(enckey) == 32 { // UNencrypted key, length 32
-		logging.Warnf("WARNING!! Key file not encrypted!!\n")
-		logging.Warnf("Anyone who can read the key file can take everything!\n")
-		logging.Warnf("You should start over and use a good passphrase!\n")
+		v, p := os.LookupEnv("LIT_KEYFILE_WARN")
```

Comment:
I think env vars are bad and no one is going to set them anyway. I don't see why we need to suppress this error. Lets print it out and if the user wants to act on it, he can. The logging level is on `Warning` anyway, so they can suppress it if they want to by setting an appropriate log level.

Reply:
Right now as far as I know there's no way for the user to generate an unencrypted keyfile, so if it's encrypted then they definitely did it intentionally and know what they're doing.  I had this so that when I'm running the test framework with `LIT_OUTPUT_SHOW=1` we decrease the noise that's printed.  (That envvar that's picked up by the testing framework.)

Really a lot of these should be config options but I don't want to make them CLI args since that's even more intrusive.  So we should push setting that up to if/when we have a config that's more than just another place for CLI args.